neqo-transport/src/connection.rs

```diff
@@ -53,6 +53,8 @@ const LOCAL_MAX_DATA: u64 = 0x3FFF_FFFF_FFFF_FFFF; // 2^62-1
 
 const MIN_CC_WINDOW: usize = 0x200; // let's not send packets smaller than 512
 
+const GRANULARITY: u64 = 1; // timer granularity in ms
```

Comment:
There is already a constant in recovery.rs.  You can make that public and pull it in.  Sorry, I got my files mixed up.

Reply:
Oh, thanks! Added. Had to cast it.